src/main/java/org/folio/clients/CirculationClient.java

```diff
@@ -65,7 +70,12 @@
     }
 
     CompositeFuture.all(futures)
-        .onSuccess(updatedInstances -> promise.complete(updatedInstances.result().list()))
+        .onSuccess(updatedInstances -> {
+          List<InventoryHoldingsAndItems> combinedList = new ArrayList<>();
+          combinedList.addAll(updatedInstances.result().list());
+          combinedList.addAll(instancesNoItems);
+          promise.complete(combinedList);
```

Comment:
Could an alternative for this be to not remove the instances without items from the list in the first place?